src/HdbConfigurationManager.cpp

```diff
@@ -2056,19 +2029,38 @@ void HdbConfigurationManager::attribute_add()
 		error_desc << "Failed to configure the attribute: " << signame << " in the database" << endl;
 		Tango::Except::re_throw_exception(e, "Attribute Configuration Query Error", error_desc.str(), __func__, Tango::ERR);
 	}
+	*/
```

Comment:
It looks like lines 2016 to 2032 can be removed now?

Reply:
yes indeed.